mock/mock_ip.go

```diff
@@ -4,48 +4,51 @@ import (
 	"errors"
 	"reflect"
 
-	"github.com/PabloPie/Gandi-Go/hosting"
+	"../hosting"
+	//"github.com/PabloPie/Gandi-Go/hosting"
 )
 
 var ips = []hosting.IPAddress{
 	hosting.IPAddress{
 		ID:       1,
 		IP:       "2001:4b98:dc0:41:216:3eff:fe9b:1c39",
 		RegionID: 3,
-		Version:  "6",
+		Version:  6,
 		VM:       1,
 		State:    "created",
 	},
 	hosting.IPAddress{
 		ID:       2,
 		IP:       "46.226.108.29",
 		RegionID: 3,
-		Version:  "4",
+		Version:  4,
 		VM:       1,
 		State:    "created",
 	},
 	hosting.IPAddress{
 		ID:       3,
 		IP:       "2001:4b98:dc2:41:216:3eff:fea8:c071",
 		RegionID: 4,
-		Version:  "6",
+		Version:  6,
 		VM:       2,
 		State:    "created",
 	},
 }
 
 func hostingIfaceCreate(args []interface{}, reply interface{}) error {
-	if len(args) < 1 {
-		return errors.New("iface.create() takes 1 argument")
+	if len(args) < 2 {
```

Comment:
the number of parameters of the Gandi v4 api is only 1 (not counting the apikey which we ignore)